lib/byron/test/data/cardano-node/node.config

```diff
@@ -10,8 +10,9 @@
 
 NodeId:
 NumCoreNodes: 1
-PBftSignatureThreshold: 1000000000000000
+PBftSignatureThreshold: 1
```

Comment:
Why this change? This was actually necessary to get the self node keeps producing block alone.

Reply:
Duncan said 1 was the max value, and it seemed to work just as well with 1.

> to get the self node keeps producing block alone.

It didn't produce blocks otherwise? Now it seems to at least.